website/client/src/pages/settings/settingRows/classSetting.vue

```diff
@@ -80,6 +82,11 @@ input {
   margin-right: 2rem;
 }
 
+.currency-align {
+  display: flex;
+  align-items: center;
```

Comment:
There are helper classes for these. You don't need to add more CSS.

Reply:
Removed, but still figuring it out

Reply:
You only ended up adding `d-flex`, was `align-items: center` not necessary in the end?

Reply:
It was not!